hack/install-odf.sh

```diff
@@ -9,8 +9,22 @@ OPERATOR_SDK=${OPERATOR_SDK:-$1}
 BUNDLE_IMG=${BUNDLE_IMG:-$2}
 ODF_DEPS_CATALOG_IMG=${ODF_DEPS_CATALOG_IMG:-$3}
 CSV_NAMES=${CSV_NAMES:-$4}
+CI=${CI:-false}
 
 NAMESPACE=$(oc get ns "$INSTALL_NAMESPACE" -o jsonpath="{.metadata.name}" 2>/dev/null || true)
+
+function print_debug_logs {
+    if [ "$CI" == true ]; then
+        echo "printing debug logs"
+        oc get csv -n "$INSTALL_NAMESPACE"
+        oc get pods -n "$INSTALL_NAMESPACE"
+        oc describe csv -n "$INSTALL_NAMESPACE"
+        oc describe pods -n "$INSTALL_NAMESPACE"
+    fi
+}
```

Comment:
Why not use the CI feature to run must-gather to collect logs after tests?

Reply:
We are already collecting must gather. But having it here is useful. Because in case of failure, we see this directly on the job page instead of going here and there and downloading the must gather.

Reply:
It's simple enough, but if you wish to print to stdout, it's your call.